FsAutoComplete/CommandResponse.fs

```diff
@@ -258,7 +266,7 @@ module CommandResponse =
                   [ for o in meth.Methods do
                      let tip = TipFormatter.formatTip o.Description
                      yield {
-                       Tip = tip
+                       Tip = {Signature = fst tip.Head; Comment = snd tip.Head } 
```

Comment:
We could look to see what VFPT does. It seems like it should have exactly 1 really, but who knows.